tests/acceptance/bootstrap/SpacesTUSContext.php

```diff
@@ -148,19 +147,50 @@ public function userCreatesANewTusResourceForTheSpaceUsingTheWebdavApiWithTheseH
 	 * @throws Exception|GuzzleException
 	 */
 	private function uploadFileViaTus(string $user, string $content, string $resource, string $spaceName): void {
-		$spaceId = $this->spacesContext->getSpaceIdByName($user, $spaceName);
-		$tmpFile = $this->tusContext->writeDataToTempFile($content);
-		try {
-			$this->tusContext->uploadFileUsingTus(
-				$user,
-				\basename($tmpFile),
-				$resource,
-				$spaceId
+		$server = $this->featureContext->getCurrentServer();
+
+		//check if it is REMOTE server and Shares space
+		if ($server === 'REMOTE' && $spaceName === 'Shares') {
```

Comment:
$server is used only in the if statement
```suggestion
		if ($this->featureContext->getCurrentServer() === 'REMOTE' && $spaceName === 'Shares') {
```